Python/Setup/swix/file_associations.swr

```diff
@@ -0,0 +1,38 @@
+use vs
+
+vs.fileAssociations
+    vs.fileAssociation
+        Extension=.py
+        ProgId=VisualStudio.py.[InstanceId]
+        DefaultProgramRegistrationPath=$(VSDefaultProgramPath)
+    
+    vs.fileAssociation
+        Extension=.pyw
+        ProgId=VisualStudio.py.[InstanceId]
+        DefaultProgramRegistrationPath=$(VSDefaultProgramPath)
+    
+    vs.fileAssociation
+        Extension=.pyproj
+        ProgId=VisualStudio.Launcher.pyproj.[InstanceId]
+        PerceivedType=text
```

Comment:
It means that the items in `HKCR\SystemFileAssociations\text` also apply to this file type, which is wrong in this case. Updating this to be `ContentType=application/xml` as it should be (and moving `PercievedType=text` onto the source file types).